try:
    print("test")
    # generate an error: the variable test is not defined
    print(test)
except:
    print("Caught an exception")